drivers/display/Kconfig.sdl

```diff
@@ -0,0 +1,45 @@
+# Kconfig - SDL based emulated display configuration options
+
+#
+# Copyright (c) 2018 dXplore
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig SDL_DISPLAY
+	bool "SDL based emulated display"
+	depends on SOC_POSIX
+	default n
```

Comment:
I guess you could set this driver as default one `if DISPLAY` for native posix in https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/posix/native_posix/Kconfig.defconfig